test/unit/org/apache/cassandra/index/sai/memory/AbstractKeyKeyRangeIteratorTest.java

```diff
@@ -23,15 +23,15 @@
 
 import org.apache.cassandra.dht.Murmur3Partitioner;
 import org.apache.cassandra.index.sai.SAITester;
+import org.apache.cassandra.index.sai.iterators.KeyRangeIterator;
 import org.apache.cassandra.index.sai.utils.PrimaryKey;
 import org.apache.cassandra.index.sai.disk.v2.RowAwarePrimaryKeyFactory;
-import org.apache.cassandra.index.sai.utils.RangeIterator;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
 
 @Ignore
-public abstract class AbstractKeyRangeIteratorTest
+public abstract class AbstractKeyKeyRangeIteratorTest
```

Comment:
This one would be `AbstractKeyRangeIteratorTest`, but this one was an abstract class without known implementations? And we now have an `AbstractKeyRangeIteratorTest` class that might supersede this one.

Reply:
Good that you noticed. The usage of it was removed in https://github.com/datastax/cassandra/pull/1230 to my understanding. So I will remove this file.